packages/sdk/src/modules/did/index.ts

```diff
@@ -34,16 +34,42 @@ interface CreateDidOptions {
   customDocumentFields?: CustomDocumentFields;
 }
 
+interface ReadDidOptions {
+  name: string;
+  address?: Address;
+}
+
+interface UpdateDidOptions {
+  name: string;
+  address?: Address;
+  seed?: string;
+  customDocumentFields: CustomDocumentFields;
+}
+
+interface RemoveDidOptions {
+  name: string;
+  address?: Address;
+  seed?: string;
+}
+
 interface CreateDidResult {
   hash: CodecHash;
   unsubscribe: () => void;
 }
 
-interface ReadDidOptions {
-  name: string;
-  address?: Address;
+interface RemoveDidResult {
```

Comment:
Returned result after calling the update/remove function. Contains a log, hash, and unsubscribe. The log has a message to say that it has been updated/remove. The hash is the extrinsic hash (update to block hash?).